vita3k/codec/src/mjpeg.cpp

```diff
@@ -262,75 +347,76 @@ bool MjpegDecoderState::receive(uint8_t *data, DecoderSize *size) {
         return false;
     }
 
+    if (mcu_width == 0 || mcu_height == 0) {
+        LOG_WARN("Mjpeg MCU size is not set. Falling back to 1x1.");
+        mcu_width = mcu_height = 1;
+    }
```

Comment:
```suggestion
    if (mcu_width == 0 || mcu_height == 0) {
        LOG_WARN_ONCE("Mjpeg MCU size is not set. Falling back to 8x8.");
        mcu_width = mcu_height = 8;
    }
```
A jpeg MCU block width and height should only be 8 or 16.